packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Server/elements.tsx

```diff
@@ -0,0 +1,53 @@
+import styled, { css } from 'styled-components';
+import { Button } from '@codesandbox/common/lib/components/Button';
+import PowerIcon from 'react-icons/lib/md/power-settings-new';
+import { WorkspaceInputContainer } from '../../elements';
+
+export const SubTitle = styled.div`
+  text-transform: uppercase;
+  font-weight: 700;
+  color: ${props =>
```

Comment:
Don't forget to use @Saeris' convention here 🙂

Reply:
In the theme I'm not gonna add it because we don't type it anyway